src/main/java/org/commcare/core/parse/ParseUtils.java

```diff
@@ -6,8 +6,8 @@
 import org.javarosa.xml.util.UnfullfilledRequirementsException;
 import org.xmlpull.v1.XmlPullParserException;
 
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
+import java.nio.charset.StandardCharsets;
```

Comment:
the `StandardCharsets` import is unused, and I think we usually prefer to be implicit about imports instead of using `.*`, especially if it's only 2 classes from that package